python-lib/dku_plugin_config_loading.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+from dataiku.customrecipe import *
+import dataiku
+import pandas as pd
+from dataiku import pandasutils as pdu
+from dataiku.customrecipe import get_recipe_config
+from language_dict import SUPPORTED_LANGUAGES_SPACY
+from dku_config import DkuConfig
+
+# All plugin parameters name
+MATCHING_PARAMS = ["case_sensitivity", "lemmatization", "unicode_normalization"]
+
+
+class DkuConfigLoading:
```

Comment:
Since later on, we will have over classes to load config for other recipes, I think it would be good to break it down in:

one parent class "PluginConfigLoader"
one child class "PluginConfigLoaderOntologyTagging"

Reply:
here :  1016248